src/frontend/apps/standalone_site/src/features/ClassRoom/components/ClassRoom.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import { screen } from '@testing-library/react';
+import { render } from 'lib-tests';
+import { MemoryRouter } from 'react-router-dom';
+
+import { routes } from 'routes';
+
+import ClassRoom from './ClassRoom';
+
+jest.mock('./Read/ClassRooms', () => ({
+  __esModule: true,
+  default: () => <div>My ClassroomsRead</div>,
+}));
+
+jest.mock('./Update/ClassRoomUpdate', () => ({
+  __esModule: true,
+  default: () => <div>My ClassRoomUpdate</div>,
+}));
+
+describe('<ClassRoom/>', () => {
+  const classroomRoute = routes.CONTENTS.subRoutes.CLASSROOM;
+  const classroomPath = classroomRoute.path;
+  const classroomCreatePath = classroomRoute.subRoutes?.CREATE?.path || '';
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('render classroom', () => {
+    render(
+      <MemoryRouter initialEntries={[classroomPath]}>
+        <ClassRoom />
+      </MemoryRouter>,
+    );
```

Comment:
The render already wrap in a router the component you want to test. Here you will have a memory router inside a memory router, this is not what you want I think. Take a look of the `RouterOptions` available for this `render` function.